challenges.py

```diff
@@ -49,9 +45,9 @@
 table.add_column("Winner")
 for challenge in challengeWinnersPerChallenge:
     allChallengeWinners = tuple(challengeWinnersPerChallenge[challenge])
-    if len(allChallengeWinners) > 0:
+    if allChallengeWinners:
```

Comment:
No `len()` needed - empty containers (sets, lists, tuples, etc.) evaluate to false, while containers with anything in them evaluate to true.